### DISCUSSION TITLE
libusb 1.0.24 update formula?

### DISCUSSION THREAD

**Discussion by mvp** [Homebrew/discussions#262] [Everyday usage] [answered]
libusb-1.0.24 has been released, which has important fixes for Catalina and Big Sur: https://github.com/libusb/libusb/commit/c6a35c56016ea2a
Can someone please update brew libusb recipe from 1.0.23 to 1.0.24?
I would put PR, but don't know what kind of testing is involved here.
@fxcoudert you did it before? Thanks!

**Comment by carlocab** [comment, marked as answer]
PR opened:

https://github.com/Homebrew/homebrew-core/pull/66673/files

Next time, one way to simplify setting up the PR yourself is to do `brew bump-formula-pr --version=<new version> <formula>`. (That is literally all I did to get that PR up.)

You may need to set up a GitHub token and assign it to `HOMEBREW_GITHUB_API_TOKEN` with `export HOMEBREW_GITHUB_API_TOKEN=<token>`.

**Comment by carlocab** [comment]
The update has been merged: https://github.com/Homebrew/homebrew-core/commit/b19ea71d5555e364361c8e67dd1110d1762fef0b
